liquibase-core/src/main/java/liquibase/change/ChangeMetaData.java

```diff
@@ -137,6 +137,6 @@ public Set<String> getAppliesTo() {
     }
 
     public boolean appliesTo(DatabaseObject databaseObject) {
-        return (appliesTo != null) && appliesTo.contains(databaseObject.getObjectTypeName());
+        return appliesTo.contains(databaseObject.getObjectTypeName());
```

Comment:
Here appliesTo can be null as the cosntructor sets it to null in some circunstances. Can you review it?

Reply:
Added back this null check.